rtengine/camconst.cc

```diff
@@ -339,6 +340,17 @@ CameraConst::parseEntry(void *cJSON_, const char *make_model)
         cc->pdafOffset = ji->valueint;
     }
 
+    ji = cJSON_GetObjectItem(js, "global_green_equilibration");
+
+    if (ji) {
+        if (ji->type != cJSON_Number) {
+            fprintf(stderr, "\"global_green_equilibration\" must be a number\n");
```

Comment:
@agriggio Are you planning for something different than `0` and `1` in the future? If not, why not use a JSON boolean here?

Reply:
@Floessie I looked for it in cJSON but could not find it... perhaps I missed it?

Reply:
@agriggio Yes, it seems a bit tricky, but googling revealed [this](https://sourceforge.net/p/cjson/discussion/998969/thread/3fd954a7/#cc36/4230) and [this](https://sourceforge.net/p/cjson/patches/3/#3c31) (BTW: Simon Budig is one of the Gimp developers).